src/synth/drumModules/snareDrum.js

```diff
@@ -32,7 +32,7 @@ export default function (audioCtx, destination, time, { level, tone, snappy }) {
   outputVCA.amplitude.value = outputLevel;
 
   // envelopes
-  const noiseEnv = new ADGenerator(LINEAR, 0.1, 75, 0, 0.5);
+  const noiseEnv = new ADGenerator(LINEAR, 0.1, 160, 0, 0.9);
```

Comment:
Is there anything in the code that mimics the subtle nuances of the bridged T-Network components used in both the Kick and Snare sounds of a real 808?  If not, I think we can agree that just setting basic ADSR envelops to certain specs probably won't do the trick. We also don't know what frequencies are used and so again I think we're guessing and need to trust our ears.  I suppose you've already looked at http://fa.utfs.org/diy/rolandtr808/roland_TR-808_service-manual.pdf
Interesting info here too:  https://www.kvraudio.com/forum/viewtopic.php?p=5778538